website/templates/project/project_header.mako

```diff
@@ -158,7 +158,7 @@
         % endif
 
         % if node['is_embargoed']:
-            <div class="alert alert-danger">This ${node['node_type']} is currently embargoed. It will remain private until its embargo date, ${ node['embargo_end_date'] }.</div>
+            <div class="alert alert-danger">This ${node['node_type']} registration is currently embargoed. It will remain private until its embargo end date, ${ node['embargo_end_date'] }.</div>
```

Comment:
This needs to be just `registration` not the node type as well (per our conversation)